internal/api/api_test.go

```diff
@@ -74,3 +75,50 @@ func TestMapQueryParameters(t *testing.T) {
 		})
 	}
 }
+
+var FlattenedCSVHeaderFixture = []string{
```

Comment:
@saltgen - I noticed that the same constant variable is declared separately in both the App level and the tests. While this works, it can lead to duplication and maintenance issues if the constant ever needs to change. 
Do you think that it might be cleaner and safer to define the constant once in a shared package and reuse it in both the app and tests? (example - we already have api package > common file OR one can declare specific to CSV) This keeps things DRY and ensures consistency across the codebase.

Reply:
@kgaikwad The test and app should ideally have separate objects, the one in the test is suffixed with `Fixture`.
Tests should have separate context altogether.